lib/common.c

```diff
@@ -42,6 +42,17 @@ void mem_bitwise_and(uint32_t *dst, uint32_t *mask, size_t len)
     }
 }
 
+void swap_byte_order(uint8_t *data, size_t len)
```

Comment:
I belive that `char *` is commonly preferred to pass a pointer to the beginning of data, instead of `uint8_t *`

Reply:
Done